backend/src/routes/trading.py

```
@@ -45,4 +45,26 @@ async def sell_token():
+        target_price = data.get('target_price')
+        amount_sol = data.get('amount_sol')
+        side = data.get('side', 'buy')
+        token_symbol = data.get('token_symbol')
+
+        if not token_address or target_price is None or amount_sol is None:
+            return error_response('Missing required fields for limit order', 400)
+
+        result = await trading_service.place_limit_order(token_address, target_price, amount_sol, side, token_symbol)
+        return jsonify(result)
+
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)